### PR TITLE
updated front end to work with new BE

### PR DESCRIPTION
**Pull request by MichaelHabermas**
### Description

**Invited Collaborator(s) to Review
- Added person to review relating to feature/bug
- If not available, ask another reviewer on team

**What work was done?**
- Describe in detail what's been modified/created

**Why was this work done?**
- What is the purpose of the feature?

**What feature / user story is it for?**
(Link if possible) 
**Any relevant links or images / screenshots**

### Type of change
Please delete options that are not relevant.
- New feature (non-breaking change which adds functionality)

Testing
- Has this feature been tested before conflict?
- Has this feature been tested after conflict?

Change Status
- Completed, ready to review and merge

### Checklist
- My code follows the style guidelines of this project
- I have performed a self-review of my own code
- My code has been reviewed by at least one peer
- I have commented my code, particularly in hard-to-understand areas
- I have made corresponding changes to the documentation
- My changes generate no new warnings
- There are no merge conflicts

### CODE REVIEW

File: src/components/common/MyProfileComponent.js [hunk 1]
```diff
@@ -6,6 +6,7 @@ function MyProfileComponent({ user, Role, Programs }) {
   //Be sure to pass the TitleText in quotes when you render component
   const { Text } = Typography;
   const history = useHistory();
+  console.log('user: ', user);
```
Comment (santorap) [inline src/components/common/MyProfileComponent.js]: Remove the console.log here if you can.

File: src/components/common/MyProfileComponent.js [hunk 2]
```diff
@@ -23,11 +24,11 @@ function MyProfileComponent({ user, Role, Programs }) {
         </Space>
         <Space direction="vertical" className="card-child">
           <Text type="secondary">Role</Text>
-          <Text className="card-text">{user.role}</Text>
+          <Text className="card-text">{user.provider_role_id}</Text>
         </Space>
         <Space direction="vertical" className="card-child">
           <Text type="secondary">Programs</Text>
-          <Text className="card-text">{user.programs.length}</Text>
+          {/* <Text className="card-text">{user.programs.length}</Text> */}
```
Comment (santorap) [inline src/components/common/MyProfileComponent.js:30]: I assume this is a temporary commenting out. It is fine for now, but let's make sure to revisit this once we have the new DB working!

**Review by santorap — APPROVED** [review]
Looks good with some minor revisions on some comments in the code

File: src/components/common/MyProfileComponent.js [hunk 3]
```diff
@@ -6,7 +6,6 @@ function MyProfileComponent({ user, Role, Programs }) {
   //Be sure to pass the TitleText in quotes when you render component
   const { Text } = Typography;
   const history = useHistory();
-  console.log('user: ', user);
```
Comment (santorap) [inline src/components/common/MyProfileComponent.js]: Thanks!

**Review by santorap — APPROVED** [review]
Great

**Review by blackenjoy27 — APPROVED** [review]
I have pair program with Michael. I see all the changes and they all make senses. Good work.

**Review by ahmedseragcodes — APPROVED** [review]
Changes make sense, see no errors, approved

**Pull request merged**